### PR TITLE
Desktop responsive web

### PR DESCRIPTION
**Pull request by ijosue16**
The complete website with responsive queries based on the device width viewport

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -363,8 +335,255 @@ p.paragraph {
...
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .body {
+    flex: none;
+    width: 100%;
+  }
+
```
Comment (emmiiorji) [inline styles.css:413]: Your design looks great, you have put a lot of work into this. However, there seems to be an overflow on the page. For a better user experience on the website, kindly fix this.

image

File: styles.css [hunk 2]
```diff
@@ -363,8 +335,255 @@ p.paragraph {
...
+  .indicator {
+    visibility: hidden;
+  }
+
+  .mywork-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    margin: 0;
+  }
```
Comment (emmiiorji) [inline styles.css:440]: You seem to have less passing in the works container than appears in the design. I recommend that you implement the given design as closely as possible.

Your design:
image


Figma design:
image

File: styles.css [hunk 3]
```diff
@@ -363,8 +335,255 @@ p.paragraph {
...
+
+  .mywork {
+    width: 100%;
+    padding: 0;
+  }
+
+  .recents-page {
+    justify-content: center;
+  }
```
Comment (emmiiorji) [inline styles.css:430]: A look at the hero page reveals that there's more margin between the background image and the recent-works heading than you have represented. Kindly adjust this if you may, to match the design template.

Your design:
image


Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @ijosue16, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Highlights✔

- Linter checks are passing
- Adhered to GitHub flow
- Repo is professional


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -363,8 +335,255 @@ p.paragraph {
...
+    margin-top: 8rem;
+    margin-bottom: 25rem;
+  }
+
+  .background-desktop {
+    background-image: url(./tools/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: initial;
+    width: 100%;
+  }
```
Comment (emmiiorji) [inline styles.css:397]: For the best experience on your website, kindly ensure that the mobile version is responsive until the desktop screen size.

image

File: styles.css [hunk 5]
```diff
@@ -363,8 +334,270 @@ p.paragraph {
...
+
+  .background-desktop {
+    background-image: url(./tools/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: initial;
+    width: 100%;
+    align-items: center;
+  }
```
Comment (OyePriscilla) [inline styles.css:406]: - [ ] Please kindly take note that your project is not well aligned (@min-width: 768px), kindly work  more on this as the previous reviewer pointed out.
- [ ] Moreso, the background and all other elements are not well positioned. Kindly make necessary adjustments.

## Your Design 
127 0 0 1_5500_Mobile-portofolio_index html (1)

127 0 0 1_5500_Mobile-portofolio_index html (2)

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ijosue16   😃 , 

Good job so far!
There are little issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Descriptive commit messages 😄 
LInters checks passed ✔️ 
GitHub flow observed ✔️ 

## Required Changes ♻️
Kindly check the inline comments below for the required changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 5 above)
Comment (ijosue16) [reply styles.css:406]: @OyePriscilla They told us to do media queries of devices that have a viewport width which is above 768px
so I don't see where it is badly aligned as I have done what they told us in the challenge

**Comment by OyePriscilla** [comment]
@ijosue16, kindly check your work @768px, it's not yet aligned properly. It is not so bad actually but there is room for improvement.

**Pull request closed without merging**